### ISSUE TITLE
Mirrors/Alternative Formats

### ISSUE THREAD

**Issue by jpoles1** [lxaw/ComprehensiveFoodDatabase#6]
Hello, I wanted to thank the team for their work in constructing this dataset! I'm exploring using this dataset for a recipe management project I'm building, but had a lot of trouble downloading/importing the provided .sql data.

1) It seems the Git LFS quota has been reached so I was unable to clone the dataset, I had to fork, enable lfs in archive, archive, and then download the archive to get the dataset. Users hoping to access the data may have luck by downloading a zip of my archived repo here: https://github.com/jpoles1/ComprehensiveFoodDatabase
2) I wanted to use the data in a SQLite database, but the sql file is not compatible. It was too large/failed to convert when I tried to use available conversion solutions. I eventually imported into MySQL and exported CSV. I've provided a copy of the CSV/SQlite versions of the database here: https://drive.google.com/file/d/1xgum9v9HLKbWyYU_8r6Lcjh_gx-GQkQ1/view?usp=drive_link

I'm not sure if there's a better way to continue providing this dataset to the community, so wanted to bring up the issue here. 

PS: I've also written up a [quick tutorial](https://blog.jpoles1.com/archives/277) on getting full text search working using SQLite, thought it might be helpful for giving new users a place to start for a barebones implementation that can be used in many different applications.

**Comment by lxaw** [comment]
Thank you very much for your comments! Yes, I did recently receive a message about LFS storage. I will soon add your contributions to the repository.

I wonder if Mega or another file sharing site would be better to store the dataset. Regardless, I will add your version within the month. 

Furthermore, thank you for the full text example. Would it be okay to for now link to your post about the full text search?

Please feel free to let us know if there are any other troubles. Hope that the dataset is of use!

**Comment by jpoles1** [comment]
You are definitely welcome to link to the post! I'm currently also developing a slight expansion to the dataset by adding on the [Canadian Nutrient File](https://www.canada.ca/en/health-canada/services/food-nutrition/healthy-eating/nutrient-data/canadian-nutrient-file-2015-download-files.html) data which has a bit better coverage for raw cooking ingredients (important for my current use case). I'll post more and share my final data shortly once I've gotten everything all tuned in!

**Comment by lxaw** [comment]
That would be great! Perhaps we can add data from multiple countries into this repository. Thanks again!

**Comment by jpoles1** [comment]
Alrighty, here's what I've got with the canadian data:
- Canadian Nutrition Data (canfood.sqlite): https://drive.google.com/file/d/1noLSp_v6_UxXrtt-4qzf2R8-vOz9TCuR/view?usp=sharing
- SQLite databse with fulltext search now including candian data: https://drive.google.com/file/d/1zZwxqLsKkqOO5VV-qsuIIZe3A5OW2W1i/view
- Blog post w/ methodology: https://blog.jpoles1.com/archives/288

**Comment by lxaw** [comment]
@jpoles1 Great, thank you for the contribution. I will add your files to the MEGA link that the project is now going to be using. 

Addressing the original first two comments:

1. "It seems the Git LFS quota has been reached..." 
 -> I have since added a link to a shared MEGA folder that the project will be hosted on.
3. "I wanted to use the data in a SQLite database..." 
-> I have provided your copy (thanks for that!) as well to the MEGA folder.

I have also added your blog posts to the repository, and created a "Special Thanks" section for mentioned help from people like you.

I will close this issue unless you have any more thoughts. Thanks again!
